src/helpers/rendering.ts

```diff
@@ -11,11 +11,19 @@ export function drawHighlight(
   }
 
   const { ctx } = renderingContext;
-  ctx.strokeStyle = highlight.color;
-  /** + 0.5 offset to have sharp lines. See comment in {@link RendererPlugin#drawBorders} for more details */
-  ctx.strokeRect(x + 0.5, y + 0.5, width, height);
-  ctx.globalCompositeOperation = "source-over";
+  if (!highlight.noBorder) {
+    ctx.strokeStyle = highlight.color;
+    if (highlight.thinLine) {
+      ctx.lineWidth = 1;
+      ctx.strokeRect(x, y, width, height);
+    } else {
+      ctx.lineWidth = 2;
+      /** + 0.5 offset to have sharp lines. See comment in {@link RendererPlugin#drawBorder} for more details */
+      ctx.strokeRect(x + 0.5, y + 0.5, width, height);
+    }
+  }
   if (!highlight.noFill) {
+    ctx.globalCompositeOperation = "source-over";
```

Comment:
what does this do ?

Reply:
Nothing, that's the default. It says "draw over exisiting things in canvas".

I moved the line, but we can remove it altogether